data-model-solr/src/main/java/org/mousephenotype/cda/solr/service/ExpressionService.java

```diff
@@ -662,6 +663,85 @@ public Model getExpressionDataForGene(String acc, Model model, boolean embryo) t
 
 	}
 
+	
+	public List<AnatomyPageTableRow> getLacZDataForAnatomy(String anatomyId,List<String> anatomyTerms, List<String> phenotypingCenter,
+			List<String> procedure, List<String> paramAssoc, String baseUrl) throws SolrServerException{
+		System.out.println("calling -------------------------------------getLacZDataForAnatomy");
```

Comment:
I don't mind either way.

On 8 July 2016 at 11:06, Ilinca Tudose <notifications@github.com> wrote:

> In
> data-model-solr/src/main/java/org/mousephenotype/cda/solr/service/ExpressionService.java
> <https://github.com/mpi2/PhenotypeData/pull/20#discussion_r70052438>:
>
> > @@ -662,6 +663,85 @@ public Model getExpressionDataForGene(String acc, Model model, boolean embryo) t
> >
> >  	}
> >
> > +	
> > +	public List<AnatomyPageTableRow> getLacZDataForAnatomy(String anatomyId,List<String> anatomyTerms, List<String> phenotypingCenter,
> > +			List<String> procedure, List<String> paramAssoc, String baseUrl) throws SolrServerException{
> > +		System.out.println("calling -------------------------------------getLacZDataForAnatomy");
>
> Do you want to keep the sysouts?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mpi2/PhenotypeData/pull/20/files/6062ad2b834b90c987027cade3a9aa838106bc2b#r70052438>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABvr3Y1Yt8iDzllk4m9MzQBaB5jleUeNks5qTiEmgaJpZM4JH48s>
> .
>



-- 
--------
Jonathan Warren